### PR TITLE
Dl 835

### PR DESCRIPTION
**Pull request by abdi1994**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -2225,6 +2225,19 @@ page.iht.iv.failure.technicalIssue.tryAgain = Try again now or in 24 hours by si
 page.iht.iv.failure.2fa.heading = You have not been signed in
 page.iht.iv.failure.2fa.p1 = You need to set up 2-step verification so you can sign in securely.
 
+error.registration.serviceUnavailable.p1 = Wait a few seconds and try again.
+error.registration.serviceUnavailable.p2 = If you see this message several times, you will need to sign out and register again later at <a id="registrationChecklistLink" href="{0}">https://www.tax.service.gov.uk/inheritance-tax/registration/registration-checklist</a> (save this link).
+error.registration.serviceUnavailable.p3 = You can choose to report the estate value using the <a id="paperFormLink" href="{0}">IHT205 paper form</a> instead
```
Comment (paulanderson1234) [inline conf/messages.en]: missing full stop at end of message

File: conf/messages.cy [hunk 2]
```diff
@@ -962,6 +959,8 @@ global.error.pageNotFound404.title=Heb ddod o hyd i''r dudalen – 404
 site.link.giveDetails=Rhowch fanylion
 page.iht.application.debts.mortgage.inputText.value=Swm y morgais
 page.iht.questionnaire.easy-to-use.very-difficult=Anodd iawn
+iht.registration.timeout.p1=Ni wnaethom gadw'ch atebion.
```
Comment (paulanderson1234) [inline conf/messages.cy]: Need to use double single quotes (escape) to render correctly in Welsh - currently displays without any quote

File: test/iht/testhelpers/MockObjectBuilder.scala [hunk 3]
```diff
@@ -148,10 +148,16 @@ object MockObjectBuilder {
     * Creates mock to getCaseDetails using IhtConnector
     */
 
+//  def createMockToGetCaseDetails(ihtConnector: IhtConnector,
```
Comment (jameshforster) [inline test/iht/testhelpers/MockObjectBuilder.scala]: Delete the comment please

File: conf/messages.cy [hunk 4]
```diff
@@ -1146,10 +1149,14 @@ page.iht.registration.checklist.deceased.label5=y cyfeiriad hysbys diwethaf ar g
...
 page.iht.application.property.value.browserTitle=Gwerth yr eiddo
 page.iht.application.tnrb.giftsWithReservationOfBenefit.browserTitle=Math o roddion a roddwyd i ffwrdd
 page.iht.application.overview.title=Mae''r broses o wneud cais ar gyfer {0} nawr yn {1}
 iht.gov.url=https://www.gov.uk/inheritance-tax
+page.iht.sign.out=Allgofnodi
+error.registration.serviceUnavailable.p2=Os gwelwch y neges hon sawl gwaith, bydd yn rhaid i chi allgofnodi a chofrestru eto nes ymlaen yn <a id="registrationChecklistLink" href="{0}">https://www.tax.service.gov.uk/inheritance-tax/registration/registration-checklist</a> (dylech gadw'r cysylltiad hwn).
+error.registration.serviceUnavailable.p3=Gallwch ddewis rhoi gwybod am werth yr ystâd gan ddefnyddio'r ffurflen <a id="registrationChecklistLink" https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/459774/IHT205_2011__Cymraeg.pdf">bapur IHT205</a> yn lle hynny.
```
Comment (paulanderson1234) [inline conf/messages.cy]: According to ggogle welsh document the link that should be underlined includes 'ffurflen bapur IHT205' this needs to be reviewed and amended if required to match welsh document attached to task

File: conf/messages.cy [hunk 5]
```diff
@@ -1146,10 +1149,14 @@ page.iht.registration.checklist.deceased.label5=y cyfeiriad hysbys diwethaf ar g
 iht.estateReport.declaration.dontTellHMRC=ddim yn rhoi gwybod i CThEM, o fewn 6 mis, am unrhyw newidiadau i''r ystâd sy''n cynyddu''r gwerth dros y trothwy ar gyfer Treth Etifeddiant
 page.iht.application.assets.household.overview.deceased.value.screenreader.link.change=Newid yr ateb ar gyfer gwerth yr eitemau i''r tŷ ac eitemau personol sydd yn enw {0}
 page.iht.questionnaire.easy-to-use.easy=Hawdd
+error.registration.systemError.p1=Gallwch roi cynnig arall ar fewngofnodi i'r gwasanaeth nes ymlaen yn <a id="estateReportLink" href="https://www.tax.service.gov.uk/inheritance-tax/estate-report">https://www.tax.service.gov.uk/inheritance-tax/estate-report</a> (dylech gadw'r cysylltiad hwn).
```
Comment (paulanderson1234) [inline conf/messages.cy]: Should this link not be for error.application.systemError.p1 to match the one held in message.en

In addition the Welsh translation should be updated as this does not translate to the amended version in English now that we have changed the message to 'Try again later to submit your report at ' as opposed to signing in logging into the service

File: conf/messages.cy [hunk 6]
```diff
@@ -962,6 +959,8 @@ global.error.pageNotFound404.title=Heb ddod o hyd i''r dudalen – 404
 site.link.giveDetails=Rhowch fanylion
 page.iht.application.debts.mortgage.inputText.value=Swm y morgais
 page.iht.questionnaire.easy-to-use.very-difficult=Anodd iawn
+iht.registration.timeout.p1=Ni wnaethom gadw'ch atebion.
+iht.application.timeout.p1=Gwnaethom gadw'ch cynnydd ar yr adroddiad ynghylch yr ystâd.
```
Comment (paulanderson1234) [inline conf/messages.cy]: Need to use double single quotes (escape) to render correctly in Welsh - currently displays without any quote

**Pull request merged**